contracts/programs/ocr2/src/lib.rs

```diff
@@ -656,15 +656,15 @@ fn transmit_impl<'info>(ctx: Context<Transmit<'info>>, data: &[u8]) -> ProgramRe
     require!(raw_signatures.len() % SIGNATURE_LEN == 0, InvalidInput);
     let signature_count = raw_signatures.len() / SIGNATURE_LEN;
     require!(
-        signature_count == 3 * usize::from(config.f) + 1,
+        signature_count == usize::from(config.f) + 1,
         WrongNumberOfSignatures
     );
     let raw_signatures = raw_signatures
         .chunks(SIGNATURE_LEN)
         .map(|raw| raw.split_last());
 
     // Verify signatures attached to report
-    let hash = hash::hashv(&[raw_report, report_context]).to_bytes();
+    let hash = hash::hashv(&[&[raw_report.len() as u8], raw_report, report_context]).to_bytes();
```

Comment:
offchain report signing also uses report len in hash

Reply:
@archseer just to double-check, even if report size is fixed length on Solana we'll leave it in to keep it consistent across chains, right?